src/game/construct_types.h

```diff
@@ -3,4 +3,17 @@
 // Type of construct built on tile.
 enum ConstructType {
     CHEST, REMOVE, NONE
-};
\ No newline at end of file
+};
+
+struct ConstructData {
+    ConstructType type;
+    int cost;
+    std::string name;
+    std::string description;
+    std::string image;
+};
+
+namespace Constructs {
+    static ConstructData chest = { ConstructType::CHEST, 50, "Chest", "A treasure chest full of gold.", "test.png" };
+    static ConstructData remove = { ConstructType::REMOVE, 0, "Remove", "Remove a block.", "Tomato.jpg" };
```

Comment:
This looks very satisfactory. An alternative is to parse these construct data from file in whatever format you like. [We can do this, too](http://gameprogrammingpatterns.com/prototype.html#prototypes-for-data-modeling)